app/components/projects/delete_list_modal_component.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= render(Primer::Alpha::Dialog.new(title: t(:'projects.lists.delete_modal.title'),
+                                     size: :large,
+                                     id: MODAL_ID,
+                                     data: { 'test-selector': MODAL_ID })) do |d| %>
+  <% d.with_header(variant: :large, mb: 2) %>
+  <% d.with_body { t(:'projects.lists.delete_modal.text') } %>
+
+  <%= render(Primer::Alpha::Dialog::Footer.new) do %>
```

Comment:
I believe rendering the footer this way messes with the slot mechanism for the Primer Component. Using `d.with_footer` ensures the slot is properly registered.

By inspecting the component's source code, it seems registering anything other than slots overrides the component's body.

```suggestion
  <% d.with_footer do %>
```

Reply:
<img width="760" alt="image" src="https://github.com/opf/openproject/assets/61627014/1ce1062f-8dbb-4c5c-969a-06db59157b6c">

Reply:
Thanks for catching this, @aaron-contreras. I use the render before also for the body and then found it better to use the slots but apparently wasn't thorough enough when making the change.